### PR TITLE
Use svg for some icons, refs 2736

### PR DESCRIPTION
**Pull request by mwjames**
This PR is made in reference to: #2736

This PR addresses or contains:

- Instead of base64/png icons use svg instead
- URLs created with the help of https://yoksel.github.io/url-encoder/, http://editor.method.ac/

This PR includes:
- [x] Tests (unit/integration)
- [x] CI build passed

### CODE REVIEW

File: res/smw/ext.smw.css [hunk 1]
```diff
@@ -188,54 +188,61 @@ span.smwsearchicon {
...
... xmlns='http://www.w3.org/2000/svg' version='1.1' height='15' width='15' viewBox='0 0 22 22'%3E%3Cpath d='m132.77 118.03l-27.945-27.945c6.735-9.722 10.1-20.559 10.1-32.508 0-7.767-1.508-15.195-4.523-22.283-3.01-7.089-7.088-13.199-12.221-18.332-5.133-5.133-11.242-9.207-18.33-12.221-7.09-3.01-14.518-4.522-22.285-4.522-7.767 0-15.195 1.507-22.283 4.522-7.089 3.01-13.199 7.088-18.332 12.221-5.133 5.133-9.207 11.244-12.221 18.332-3.01 7.089-4.522 14.516-4.522 22.283 0 7.767 1.507 15.193 4.522 22.283 3.01 7.088 7.088 13.197 12.221 18.33 5.133 5.134 11.244 9.207 18.332 12.222 7.089 3.02 14.516 4.522 22.283 4.522 11.951 0 22.787-3.369 32.509-10.1l27.945 27.863c1.955 2.064 4.397 3.096 7.332 3.096 2.824 0 5.27-1.032 7.332-3.096 2.064-2.063 3.096-4.508 3.096-7.332.0001-2.877-1-5.322-3.01-7.331m-49.41-34.668c-7.143 7.143-15.738 10.714-25.787 10.714-10.05 0-18.643-3.572-25.786-10.714-7.143-7.143-10.714-15.737-10.714-25.786 0-10.05 3.572-18.644 10.714-25.786 7.142-7.143 15.738-10.714 25.786-10.714 10.05 0 18.643 3.572 25.787 10.714 7.143 7.142 10.715 15.738 10.715 25.786 0 10.05-3.573 18.643-10.715 25.786' transform='matrix%28.11417.00745-.00745.11417 3.93 2.548%29' fill='%23ccc'/%3E%3C/svg%3E")
```
Comment (Nikerabbit) [inline res/smw/ext.smw.css]: Embed doesn't do anything for data urls, as far as I know. It would embed svg files reference in a relative url though.

Comment (mwjames) [reply res/smw/ext.smw.css]: @kghbln Maybe a follow-up when addressing #2736?

Comment (kghbln) [reply res/smw/ext.smw.css]: I have added a note https://github.com/SemanticMediaWiki/SemanticMediaWiki/issues/2736#issuecomment-350847919 for tomorrow

Comment (kghbln) [reply res/smw/ext.smw.css]: Admittedly I have trouble to understand what this is about. Adding an additional svg file for the case of embed usage? In this case I do not understand the point of having data uri's in the first place.

Comment (mwjames) [reply res/smw/ext.smw.css]: If I interpret the comment correctly then a data URL doesn't require a `/* @embed */` annotation (which if at all is only relevant to the ResourceLoader). So while  `/* @embed */` works on images, it seems it is not required for those data-URI related objects.

Comment (kghbln) [reply res/smw/ext.smw.css]: > So while `/* @embed */` works on images, it seems it is not required for those data-URI related objects.

This was probably too easy. I searched my butt off to find something related to data-URIs and embed but did not find anything. So I started to wildly guess. Indeed `@embed` is for images.

**Pull request merged**